MonoGame.Framework/BoundingBox.cs

```diff
@@ -29,6 +41,11 @@ public struct BoundingBox : IEquatable<BoundingBox>
 
         #region Public Constructors
 
+        /// <summary>
+        /// Creates an axis aligned bounding box defining minimum and maximum edge.
+        /// </summary>
+        /// <param name="min">Minimum edge.</param>

```

Comment:
Yes, you are right. I dunno if @tomspilman finds it good to fix this when there is already a working pull request state.